includes/rest/class-interaction-controller.php

```diff
@@ -43,12 +43,18 @@ public function register_routes() {
 					'callback'            => array( $this, 'get_item' ),
 					'permission_callback' => '__return_true',
 					'args'                => array(
-						'uri' => array(
+						'uri'    => array(
 							'description'       => 'The URI or webfinger ID of the object to interact with.',
 							'type'              => 'string',
 							'required'          => true,
 							'sanitize_callback' => array( $this, 'sanitize_uri' ),
 						),
+						'intent' => array(
+							'default'     => '',
+							'description' => 'The intent of the interaction, e.g., follow, reply, import.',
+							'type'        => 'string',
+							'required'    => false,
```

Comment:
```suggestion
							'description' => 'The intent of the interaction, e.g., follow, reply, import.',
							'type'        => 'string',
```
These should be defaults that can be omitted.

Should we add an `enum` parameter that limits this to Activity types? Or just the two that we currently support?
```php

							'enum'        => array_map( 'strtolower', Activity::TYPES ),
```

Reply:
Something is off with the suggested change. It's suppose to remove `default` and `required`.